onnxruntime/test/optimizer/ensure_unique_dq_for_node_unit_test.cc

```diff
@@ -124,7 +124,9 @@ TEST(EnsureUniqueDQForNodeUnitTests, DQSharedAmongNodes) {
 
   // expected count = one for each explicit consumer node (3), reusing the original one = 3
   RunEnsureUniqueDQForNodeUnitTest(config, 3, false /*use_ms_domain_qdq_ops*/);
+#if !defined(DISABLE_CONTRIB_OPS)
   RunEnsureUniqueDQForNodeUnitTest(config, 3, true /*use_ms_domain_qdq_ops*/);
+#endif
```

Comment:
nit: if RunEnsureUniqueDQForNodeUnitTest internally ran both the true/false tests the check on DISABLE_CONTRIB_OPS could be in one place inside that function.